### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by c1phani1simha** [MDAnalysis/mdanalysis#4416] [GSoC Discussions]
Greetings!!

I'm Phani Simha from IIT-BHU,India. I am currently pursuing my undergraduate degree in Pharmaceutical Sciences with a minor in Computer Sciences. I've had expertise in MERN stack development for 1.5 years, and I've also been learning Python, NumPy, Pandas, and Matplotlib for a few weeks. I am looking forward to be a part of MDAnalysis GSoc'24. I am also familiar with Molecular docking and related things through my academics previously. So, I am writing this to show my deep interest towards MDAnalysis, Looking forward to contribute!!

Please comment with helper guide!!
Thank you in advance,

**Comment by jennaswa** [comment]
Hi @c1phani1simha!

Welcome to MDAnalysis, and thank you for expressing your interest in the project! While we are planning to apply for the GSoC 2024 program, it is not guaranteed that we will be selected by Google as an organization. No matter what happens, we of course welcome you to start [learning MDAnalysis](https://www.mdanalysis.org/pages/learning_MDAnalysis/) and contributing outside of GSoC!

MDAnalysis is a Python package for analyzing molecular simulations data. We have a few [introductory videos](https://www.mdanalysis.org/pages/learning_MDAnalysis/#introductory) that can give you an idea of what problems MDAnalysis is solving. If you haven't used MDAnalysis before, the best way to get started is to get your hands dirty following the [MDAnalysis User Guide](https://userguide.mdanalysis.org/stable/index.html). We suggest starting with [installing](https://userguide.mdanalysis.org/stable/installation.html) the MDAnalysis package and going through the [Quick Start Guide](https://userguide.mdanalysis.org/stable/examples/quickstart.html). Other sections of the User Guide go into many more details about the different capabilities of MDAnalysis. 

Once you are a bit familiar with the MDAnalysis package, you can look at the sections of the User Guide explaining [how to contribute](https://userguide.mdanalysis.org/stable/contributing.html). There are [detailed explanations](https://userguide.mdanalysis.org/stable/contributing_code.html) on how to set up a developer environment and how to contribute to the MDAnalysis codebase. This is a great starting point for contributing to MDAnalysis and trying to solve some of the open issues.

Hopefully this will help you get started, but please don't hesitate to reach out if you have any questions.

We look forward to your contribution.
